integration/metric.go

```diff
@@ -140,22 +141,34 @@ func (ds *StoreWithMetrics) meter(refresh time.Duration) {
 	errc <- merr
 }
 
+var tmpDbNameMask = regexp.MustCompile("^([A-z]+)(-[0-9]+)$")
+
+func genericNameOfTmpDB(name string) string {
+	match := tmpDbNameMask.FindStringSubmatch(name)
+	if len(match) == 3 {
+		return match[1] + "es"
```

Comment:
Ending with "es" looks a bit strange for `gossipes` and `epoches`.

We might include all (gossip-, lachesis- and  epoch-) into `epochs` like before?

Reply:
How about gossip-tmp, lachesis-tmp and epoch-tmp ?

Reply:
@rus-alex I would say it is more like "gossip-epoch", "lachesis-epoch" and "epoch".
But I like the idea of one "epochs" generic name more.

Reply:
@hkalina , but `gossip-%d` and `lachesis-%d` are not the same DB for "legacy-ldb" and "legacy-pbl" layouts. So we should have different metrics for each, not a common "epochs".
I agree that we mostly use "ldb-1" and "pbl-1" now. But it may be changed later.